qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java

```diff
@@ -79,7 +79,7 @@ void shouldResumeExporting() {
     final var actuator = getActuator();
     actuator.pause();
 
-    startProcess();
+    client.newPublishMessageCommand().messageName("Test").correlationKey("1").messageId("1").send();
```

Comment:
```suggestion
    client.newPublishMessageCommand().messageName("Test").correlationKey("1").messageId("1").send().join();
```

Wait for the future to make it deterministic.